packages/docs/playwright/with-metamask.test.ts

```diff
@@ -85,6 +86,25 @@ describe(`Browser: ${browserType.name()} with Metamask`, () => {
         expect(await page.isVisible(`//*[text()='Current chain: ' and text()='5']`)).to.be.true
       })
     })
+
+    it('Add new network to Metamask', async () => {
+      await page.goto(`${baseUrl}Guides/Transactions/Switching%20Networks`)
+
+      await waitForExpect(async () => {
+        expect(await page.isVisible(`//*[text()='Current chain: ' and text()='1']`)).to.be.true
+      })
+
+      const popupPromise = waitForPopup(context)
+      await page.click(XPath.text('button', 'Switch to Optimism'))
+      const popupPage = await popupPromise
+      await popupPage.click(XPath.text('a', 'View all'))
+
+      expect(await popupPage.isVisible(`//*[text()='${Optimism.chainName}']`)).to.be.true
+      expect(await popupPage.isVisible(`//*[text()='${Optimism.rpcUrl}']`)).to.be.true
+      expect(await popupPage.isVisible(`//*[text()='${Optimism.chainId}']`)).to.be.true
+      expect(await popupPage.isVisible(`//*[text()='${Optimism.nativeCurrency.symbol}']`)).to.be.true
+      expect(await popupPage.isVisible(`//*[text()='${Optimism.blockExplorerUrl}']`)).to.be.true
```

Comment:
How about adding - expect that "the data from this chain does not match metamask records" is not visible - I don't remember the message exactly, but you can recreate it by temporarily changing e.g. currency symbol